doc/manuals/admin/database_model.md

```diff
@@ -307,6 +307,10 @@ Fields:
 -   **qs**: optional field to store the query parameters keymap for notification customization functionality in NGSIv2.
 -   **method**: optional field to store the HTTP method for notification customization functionality in NGSIv2.
 -   **payload**: optional field to store the payload for notification customization functionality in NGSIv2.
+-   **lastFailure**: the time when last notification failure occurred.
```

Comment:
After today skype talk, I understand that -1 is no longer a right value. Although the code change will be done in a next PR I'll suggest to adjust notificaiton in this one in order not forget it.